README.md

```diff
@@ -24,6 +24,19 @@ The connectors can be used to build end-to-end stream processing pipelines (see
 
   - Table API support to access Pravega Streams for both **Batch** and **Streaming** use case.
 
+## Compatibility Matrix
+The [master](https://github.com/pravega/flink-connectors) branch will always have the most recent
+supported versions of Flink and Pravega.
+
+| Git Branch | Pravega Version | Java Version To Build Connector | Java Version To Run Connector | Flink Version | Status | Artifact Link |
+|-------------------------------------------------------------------------------------|------|---------|--------------|------|-------------------|----------------------------------------------------------------------------------------|
+| [master](https://github.com/pravega/flink-connectors)                               | 0.10 | Java 11 | Java 8 or 11 | 1.11 | Under Development | http://oss.jfrog.org/jfrog-dependencies/io/pravega/pravega-connectors-flink-1.11_2.12/ |
```

Comment:
Sorry I just merged the 1.12 upgrade PR, so this Flink version can update to 1.12 now
```suggestion
| [master](https://github.com/pravega/flink-connectors)                               | 0.10 | Java 11 | Java 8 or 11 | 1.12 | Under Development | http://oss.jfrog.org/jfrog-dependencies/io/pravega/pravega-connectors-flink-1.12_2.12/ |
```